build.sbt

```diff
@@ -254,8 +254,11 @@ lazy val logstageApiBaseMacro = inLogStage.as.module
     )
   )
 
+lazy val logstageConfig = inLogStage.as.module
+  .depends(distageConfig, distageModel, logstageApiBase)
+
 lazy val logstageApiLogger = inLogStage.as.module
-  .depends(logstageApiBaseMacro)
+  .depends(logstageApiBaseMacro, logstageConfig)
```

Comment:
Basic logstage API shouldn't depend on distage or the rest of the framework, it should be usable separately